configuration/debian/tedge_agent/postrm

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+
+
+### Remove user tedge-agent
+if getent passwd tedge-agent >/dev/null; then
+    pkill -u tedge-agent || true
+    deluser --quiet --system tedge-agent
+fi
+
```

Comment:
I think we have to remove `/etc/tedge/.agent` directory as it's owned by `tedge-agent`.